frontend/src/ui/OnetLink.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export interface OnetLinkProps {
+  socCode: string;
+}
+
+const OnetLink = (props: OnetLinkProps) => {
+  let newcode = props.socCode;
+  if (!newcode.endsWith('.00')) {
```

Comment:
"...I think we'll need to provide the soc_decimal_code in the table data to render this.

Separately, have we decided which occupation to use when a particular 6-digit SOC code has more than one decimal code?" - AB

" I did some digging and found no cases where there's a mapping of more than one english name for a given code. Also found no cases that did not end with .00. I am updating the code to handle either type of input though." - JP